src/redis_cmd.cc

```diff
@@ -1453,6 +1453,48 @@ class CommandHGetAll : public Commander {
   }
 };
 
+class CommandHRange : public Commander {
+ public:
+  Status Parse(const std::vector<std::string> &args) override {
+    if (args.size() != 6 && args.size() != 4) {
+      return Status(Status::RedisParseErr, errWrongNumOfArguments);
+    }
+    if (args.size() == 6 && Util::ToLower(args[4]) != "limit") {
+      return Status(Status::RedisInvalidCmd, errInvalidSyntax);
+    }
+    try {
+      start_ = std::stol(args[2]);
```

Comment:
> @tanruixiang The start/stop should be the string range instead of the index.

Ok, I misunderstood the meaning of start and stop. I'll revise it right away.